githubmarkdownui/job.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Optional
+
+from githubmarkdownui.emoji import Emoji
+
+
+class Job:
```

Comment:
Can you elaborate on what's this class is intended to be used in a docstring?

Reply:
Added docstring to mention its useful for displaying jobs within a build pipeline